create-custom-map-style/README.md

```diff
@@ -0,0 +1,990 @@
+
+### Amplify Setup
+1. Follow the instructions to initialize a new Amplify project [as outlined here](https://docs.amplify.aws/cli/start/workflows#initialize-new-project)
+2. For the prompts enter the following:
+	 ```
+	amplify init
+	
+	Note: It is recommended to run this command from the root of your app directory
+	? Choose your default editor: Visual Studio Code (or whatever editor you prefer)
+	? Choose the type of app that you're building javascript
+	Please tell us about your project
+	? What javascript framework are you using react
+	? Source Directory Path:  src
+	? Distribution Directory Path: build
+	? Build Command:  npm run-script build
+	? Start Command: npm run-script start
+	Using default provider awscloudformation
+
+	For more information on AWS Profiles, see:
+	https://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html
+
+	? Do you want to use an AWS profile? Yes
+	? Please choose the profile you want to use  Amplify-Default
+	 ```
+3. That leaves us with a root stack for our Amplify project in CloudFormation, now let's add the `Auth` category:
+	```
+	amplify add auth
+	
+	Using service: Cognito, provided by: awscloudformation
+	The current configured provider is Amazon Cognito.
+	Do you want to use the default authentication and security con
+	figuration? Yes, use the default configuration.
+	Successfully added resource cognito0cc0c000 locally
+
+	 Some next steps:
+	"amplify push" will build all your local backend resources and provision it in the cloud
+	"amplify publish" will build all your local backend and frontend resources (if you have 
+	hosting category added) and provision it in the cloud
+	```
+	(Note: `cognito0cc0c000` will always include a UUID generated by Amplify so substitute it for yours in the following steps)
+	
+4. The new resources have only been configured in our local environment so let's publish them to our AWS Account: 
+	```
+	amplify push
+	| Category | Resource name   | Operation | Provider plugin   |
+	| -------- | --------------- | --------- | ----------------- |
+	| Auth     | cognito0xx0x000 | Create    | awscloudformation |
+	? Are you sure you want to continue? Yes
+	```
+5. You might get an error like this:
+	```
+	CREATE_FAILED UserPoolClientLambda                                  AWS::Lambda::Function
+	      
+	Wed Apr 07 2021 16:50:55 GMT-0400 (Eastern Daylight Time) The runtime parameter of 
+	nodejs6.10 is no longer supported for creating or updating AWS Lambda functions. 
+	We recommend you use the new runtime (nodejs12.x) while creating or updating functions.
+	```
+	In which case we can manually edit the CloudFormation template generated by Amplify to replace all occurrences of `nodejs6.10` with `nodejs12` either in our IDE or with the command line as shown below:
+	```
+	cd ~/.../create-custom-map-style/amplify/backend/auth/cognito0cc0c000
+	sed -i -e 's/nodejs6.10/nodejs12.x/g' cognito0cc0c000-cloudformation-template.yml
+	cd ~/blog/create-custom-map-style
```

Comment:
Can you remove the assumption that this is being developed under `~/blog`? That won't be true for everyone.

Reply:
We put it as a prerequisite to create a workspace for the following the walk-through: https://github.com/beatleboy501/amazon-location-samples/tree/create-custom-map-style/create-custom-map-style#prerequisites

Should we change the name to something more generic or just specify "Inside the workspace you've created"?

Reply:
Probably "inside the workspace you've created" or similar